src/common/SPELLS/demonhunter.ts

```diff
@@ -158,6 +158,11 @@ const spells = {
     name: 'Immolation Aura First Strike',
     icon: 'ability_demonhunter_immolation',
   },
+  IMMOLATION_AURA_SECOND_STRIKE: {
```

Comment:
Im a little bit confused what to name this spell when looking att line 151 and 156

Reply:
Give them a name that fits what they do, I don't think first and second strike fit a proper descriptive name for the abilities.
It could be IMMOLATION_AURA_FIRST_HIT and IMMOLATION_AURA_DAMAGE or whatever those IDs actually do.

Reply:
Thanks for feedback! Changed the names and now I understand what both these spells actually do. I also noticed it didnt show up properly in Statistic - Talents. Also fixed that.